### PR TITLE
docs: Update deployment docs for Harbor 2.0

### PR DESCRIPTION
**Pull request by danielpacak**
Resolves: #115

Signed-off-by: Daniel Pacak <pacak.daniel@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,7 +86,23 @@ $ make docker-build
...
+
+In Harbor >= 2.0 Trivy can be configured as the default vulnerability scanner, therefore you can install it with the
+official Harbor Helm chart, where `HARBOR_CHART_VERSION` >= 1.4:
+
+```
+$ helm install harbor harbor/harbor \
+    --version=$HARBOR_CHART_VERSION \
+    --namespace harbor \
+    --set clair.enabled=false \
+    --set trivy.enabled-true
```
Comment (lizrice) [inline README.md:99]:
```suggestion
    --set trivy.enabled=true
```
?

File: README.md [hunk 2]
```diff
@@ -85,7 +86,23 @@ $ make docker-build
 
 ## Deployment
 
-### Kubernetes
+### Harbor 2.0 on Kubernetes
+
+In Harbor >= 2.0 Trivy can be configured as the default vulnerability scanner, therefore you can install it with the
+official Harbor Helm chart, where `HARBOR_CHART_VERSION` >= 1.4:
```
Comment (lizrice) [inline README.md:92]: Should the Helm chart version be >= 2.0?

Comment (danielpacak) [reply README.md:92]: There're two versions: app version = 2.0 and Helm chart version = 1.4

Comment (danielpacak) [reply README.md:92]:
```
$ helm search repo harbor/harbor                                                                                            [13:57:20]
NAME         	CHART VERSION	APP VERSION	DESCRIPTION                                       
harbor/harbor	1.4.0        	2.0.0      	An open source trusted cloud native registry th...
```

**Pull request merged**